README.md

```diff
@@ -0,0 +1,32 @@
+# oVirt Release
+[![Copr build status](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-4.4-snapshot/package/ovirt-release44/status_image/last_build.png)](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-4.4-snapshot/package/ovirt-release44/)
+
+Welcome to the oVirt Release source repository.
+
+This repository is hosted on [gerrit.ovirt.org:ovirt-release](https://gerrit.ovirt.org/#/admin/projects/ovirt-release)
```

Comment:
_Note_: either in this PR or another, we need to correct these parts about the Gerrit-GH relationship.

Reply:
done